include/rfbproto.h

```diff
@@ -475,7 +475,7 @@ typedef struct {
 #define rfbEncodingPointerPos      0xFFFFFF18
 
 #define rfbEncodingLastRect           0xFFFFFF20
-#define rfbEncodingNewFBSize          0xFFFFFF21
+#define rfbEncodingDesktopSize        0xFFFFFF21
```

Comment:
It's already defined as -223 in line 424, so this can just be deleted.

Reply:
I think that's added in my previous commit? (Which is overridden by the forced push)
Originally libvncserver didn't define it.

Reply:
Ahh, yes, my source of truth was my local source tree with your previous patch applied.